challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -10506,6 +10506,1096 @@
+        },
+        {
+          "text":
+            "'<code>queenPuzzle(4,4)</code> should return <code>'+JSON.stringify(results[0])+'</code>.'",
+          "testString":
+            "assert.deepEqual(queenPuzzle(4,4),results[0],'<code>queenPuzzle(4,4)</code> should return <code>'+JSON.stringify(results[0])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>queenPuzzle(3,4)</code> should return <code>'+JSON.stringify(results[1])+'</code>.'",
+          "testString":
+            "assert.deepEqual(queenPuzzle(3,4),results[1],'<code>queenPuzzle(3,4)</code> should return <code>'+JSON.stringify(results[1])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>queenPuzzle(2,4)</code> should return <code>'+JSON.stringify(results[2])+'</code>.'",
+          "testString":
+            "assert.deepEqual(queenPuzzle(2,4),results[2],'<code>queenPuzzle(2,4)</code> should return <code>'+JSON.stringify(results[2])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>queenPuzzle(6,6)</code> should return <code>'+JSON.stringify(results[3])+'</code>.'",
+          "testString":
+            "assert.deepEqual(queenPuzzle(6,6),results[3],'<code>queenPuzzle(6,6)</code> should return <code>'+JSON.stringify(results[3])+'</code>.');"
+        }
+      ],
+      "id": "5a23c84252665b21eecc7f3b",
+      "challengeType": 5,
+      "releasedOn": "July 28, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": [
+            "function queenPuzzle (rows, columns) {",
+            "  // Good luck!",
+            "}"
+          ],
+          "head": [],
+          "tail": [
+            "var results=[",
+            "[[1,3,0,2],[2,0,3,1]],",
+            "[[0,3,1],[1,3,0],[2,0,3],[3,0,2]],",
+            "[[0,2],[0,3],[1,3],[2,0],[3,0],[3,1]],",
+            "[[1,3,5,0,2,4],[2,5,1,4,0,3],[3,0,4,1,5,2],[4,2,0,5,3,1]]]"
+          ]
+        }
+      }
+    },
+    {
+      "title": "Narcissistic decimal number",
+      "description": [
+        "A <a href=\"http://mathworld.wolfram.com/NarcissisticNumber.html\">Narcissistic decimal number</a> is a non-negative integer, $ n $, that is equal to the sum of the $ m^{th} $ powers of each of the digits in the decimal representation of $n$, where $m$ is the number of digits in the decimal representation of $n$.",
+        "Narcissistic (decimal) numbers are sometimes called <b>Armstrong</b> numbers, named after Michael F. Armstrong.",
+        "if $ n $ is <b>153</b>",
+        "then $ m $, (the number of decimal digits) is <b>3</b>",
+        "we have  1<sup>3</sup> + 5<sup>3</sup> + 3<sup>3</sup> = 1 + 125 + 27 = <b>153</b>",
+        "and so <b>153</b> is a narcissistic decimal number</span>",
+        "Write a function that returns true if the given number is Narcissistic, otherwise false."
```

Comment:
"Write a function that returns true if the given number is Narcissistic. Otherwise, return false."